uber/site_sections/registration.py

```diff
@@ -801,6 +801,54 @@ def review(self, session):
                                     .order_by(Attendee.full_name).all()}
 
     @site_mappable
+    def promo(self, session, message='', **params):
+        pc = session.promo_code(params)
+        now = datetime.now(UTC)
+        today = "%d-%02d-%02d" % (now.date().year, now.date().month, now.date().day)
```

Comment:
Django can get today's date just fine using https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#now, so you likely don't need to generate this here just to pass it to the template.